### PR TITLE
[chore] fix the css & modify api url

### PR DESCRIPTION
**Pull request by dongjx**

### CODE REVIEW

File: src/components/fileUpload/DefaultFileUploadChild.tsx [hunk 1]
```diff
@@ -78,15 +75,9 @@ export const formatBytes = (bytes: number): string => {
...
+  const { documentType, baseUrl, token } = props;
 
-  const queryString = qs.stringify({
-    documentTypeCode: documentType!.code,
-    subjectId,
-    subjectType: SubjectType.Premise
-  });
-
-  fetch(`${baseUrl}/api/document-info/template?${queryString}`, {
+  fetch(`${baseUrl}/api/document-types/${documentType!.code}/template`, {
```
Comment (whipermr5) [inline src/components/fileUpload/DefaultFileUploadChild.tsx]: Maybe the fetch statement should be moved into the web project - this project is open source and should not have dependencies on the internal projects

Comment (whipermr5) [reply src/components/fileUpload/DefaultFileUploadChild.tsx]: Maybe this was the dependency that @Anddd7 remembered during tech retro :)

Comment (dongjx) [reply src/components/fileUpload/DefaultFileUploadChild.tsx]: I think what retro mentioned is that we could not put some sensitive info in this public repo.
Besides, this fileUploadComponent's all existing implements totally depend on our documentType now.

Comment (whipermr5) [reply src/components/fileUpload/DefaultFileUploadChild.tsx]: Okay, can be refactored in future

**Pull request merged**